### DISCUSSION TITLE
Adding Elestio as deployment option

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [searxng/searxng#3934] [General]
Hey team,
I am Kaiwalya, Developer Advocate at Elestio. Elestio has been providing options of fully deploying and managing SearXNG application as shown [here](https://elest.io/open-source/searxng). I think it would be a great idea if we can add it to official readme/documentation here.
🔔 In addition to this, if you are interested we provide partnership opportunities with tools we support by **revenue share** upon addition of this method in docs. If you would like to collaborate, just shoot me an email at [kaiwalya@elest.io](mailto:kaiwalya@elest.io) :)

**Comment by return42** [comment]
> I think it would be a great idea if we can add it to official readme/documentation here.

We (the SearXNG community) don't want to officially recommend third parties that we don't control / it is important to us that third parties share our idea of FOSS and privacy ... but we cannot verify that.

> if you are interested we provide partnership opportunities with tools we support by revenue share upon addition of this method in docs.

SearXNG is a project of the community, all (critical) decisions that we maintainers make we try to weigh up with the community and in the interests of the users. This process is no longer balanced if we have a financial stake.

Thank you for the offer, but unfortunately we have to decline.

----

To be honest, we don't need money, we need contributions: Contributions (e.g. pull requests) from companies and individuals are therefore always welcome; these are coordinated within the project and implemented in accordance with the interests of the community.
